<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin_Control</class>
 <widget class="QMainWindow" name="Admin_Control">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Panel</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_admin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Admin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="get_admins">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Admins</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_features">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Features</string>
    </property>
   </widget>
   <widget class="QPushButton" name="get_passwords">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Passwords</string>
    </property>
   </widget>
   <widget class="QPushButton" name="use_hash">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Use Hash</string>
    </property>
   </widget>
   <widget class="QPushButton" name="change_username">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Username</string>
    </property>
   </widget>
   <widget class="QPushButton" name="message_user">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Message User</string>
    </property>
   </widget>
   <widget class="QPushButton" name="get_permissions">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Permissions</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ban_user">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Ban User</string>
    </property>
   </widget>
   <widget class="QPushButton" name="get_features">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Features</string>
    </property>
   </widget>
   <widget class="QPushButton" name="force_password_reset">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Force Pass Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_account">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Account</string>
    </property>
   </widget>
   <widget class="QPushButton" name="force_logout">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Force Logout</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="block_user">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Block User</string>
    </property>
   </widget>
   <widget class="QPushButton" name="change_password">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="get_banned_users">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Banned Users</string>
    </property>
   </widget>
   <widget class="QPushButton" name="block_features">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Block Features</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_permissions">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Permission</string>
    </property>
   </widget>
   <widget class="QPushButton" name="get_groups">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Groups</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_group">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Group</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_group">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>370</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Group</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ban_user_group">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>450</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Ban User Group</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>490</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="kick_user">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>410</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Kick User</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>278</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="text">
    <string>Back</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Return to home page&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionSet_Admin">
   <property name="text">
    <string>Set Admin</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sets an admin account&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionGet_Admins">
   <property name="text">
    <string>Get Admins</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Gets a list of all the Admins&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionChange_Password">
   <property name="text">
    <string>Change Password</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Change a users password&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionGet_Passwords">
   <property name="text">
    <string>Get Passwords</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Gets a list of passwords&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionUse_Hash">
   <property name="text">
    <string>Use Hash</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Use a hash to login to a account&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionForce_Logout">
   <property name="text">
    <string>Force Logout</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Forces the targeted account to logout&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionRemove_Account">
   <property name="text">
    <string>Remove Account</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Removes a users account from the system&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="text">
    <string>Login</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Login to a account&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionChange_Username">
   <property name="text">
    <string>Change Username</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Changes a users username&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionMessage_User">
   <property name="text">
    <string>Message User</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Message a user&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionBlock_User">
   <property name="text">
    <string>Block User</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Blocks a user&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionGet_Features">
   <property name="text">
    <string>Get Features</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Get a list of all the features for a specific user&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionSet_Feature">
   <property name="text">
    <string>Set Feature</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Change features for a specific user&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionBlock_Features">
   <property name="text">
    <string>Block Features</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Block features for a specifc user&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionBan_User">
   <property name="text">
    <string>Ban User</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Bans a user from the system&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionGet_Banned_Users">
   <property name="text">
    <string>Get Banned Users</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Get a list of banned users&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionForce_Password_Reset">
   <property name="text">
    <string>Force Password Reset</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Forces a user to reset their password&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionGet_Permissions">
   <property name="text">
    <string>Get Permissions</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Get a list of the group permissions&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionSet_Permission">
   <property name="text">
    <string>Set Permission</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sets permissions for the group&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionGet_Groups">
   <property name="text">
    <string>Get Groups</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Get a list of all groups&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionSet_Group">
   <property name="text">
    <string>Set Group</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sets the name of the group&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionAdd_Group">
   <property name="text">
    <string>Add Group</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Adds a group to the system&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionKick_User">
   <property name="text">
    <string>Kick User</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Kicks a user from a group&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionBan_User_Group">
   <property name="text">
    <string>Ban User Group</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Bans a user from the group&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Exits the program&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>back</tabstop>
  <tabstop>set_admin</tabstop>
  <tabstop>get_admins</tabstop>
  <tabstop>change_password</tabstop>
  <tabstop>get_passwords</tabstop>
  <tabstop>use_hash</tabstop>
  <tabstop>force_logout</tabstop>
  <tabstop>remove_account</tabstop>
  <tabstop>login</tabstop>
  <tabstop>change_username</tabstop>
  <tabstop>message_user</tabstop>
  <tabstop>block_user</tabstop>
  <tabstop>get_features</tabstop>
  <tabstop>set_features</tabstop>
  <tabstop>block_features</tabstop>
  <tabstop>ban_user</tabstop>
  <tabstop>get_banned_users</tabstop>
  <tabstop>force_password_reset</tabstop>
  <tabstop>get_permissions</tabstop>
  <tabstop>set_permissions</tabstop>
  <tabstop>get_groups</tabstop>
  <tabstop>set_group</tabstop>
  <tabstop>add_group</tabstop>
  <tabstop>kick_user</tabstop>
  <tabstop>ban_user_group</tabstop>
  <tabstop>exit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
